build.sbt

```diff
@@ -102,13 +105,13 @@ imageNames in docker := {
   }
 }
 
-dockerfile in docker := NativeDockerfile(baseDirectory.value / "Dockerfile")
+docker / dockerfile := NativeDockerfile(baseDirectory.value / "Dockerfile")
 
-dockerBuildArguments in docker := Map(
+docker / dockerBuildArguments := Map(
   "VERSION" -> version.value,
 )
 
-buildOptions in docker := BuildOptions(
+docker / buildOptions := BuildOptions(
```

Comment:
is the syntax of the `/` the preferred way in Scala?    To my eyes (and this is totally personal), the `in` as an interators makes sense to me...  or am I misunderstanding the `/` operator?   Darn scala!

Reply:
This is just the new SBT syntax since 1.1.0 leading to deprecation warnings since 1.5.0 (https://www.scala-sbt.org/1.x/docs/Migrating-from-sbt-013x.html#slash)